config/initializers/omniauth.rb

```diff
@@ -0,0 +1,28 @@
+OmniAuth.config.logger = Rails.logger
+
+dfe_sign_in_issuer = ENV['DFE_SIGN_IN_ISSUER']
+dfe_sign_in_redirect_base_url = HostingEnvironment.application_url
+dfe_sign_in_identifier = ENV['DFE_SIGN_IN_CLIENT_ID']
+dfe_sign_in_secret = ENV['DFE_SIGN_IN_SECRET']
+
+dfe_sign_in_issuer_uri = dfe_sign_in_issuer.present? ? URI(dfe_sign_in_issuer) : nil
```

Comment:
Perhaps this can use the ENV var directly to avoid intermediate variables.

Reply:
yes